src/Console/ConfigurationResolver.php

```diff
@@ -847,7 +847,7 @@ static function ($path) {
             }
 
             return new \CallbackFilterIterator(
-                $nestedFinder,
+                new \IteratorIterator($nestedFinder),
```

Comment:
?

Reply:
First argument of `CallbackFilterIterator`'s constructor expects an `Iterator`, `$nestedFinder` is a `Traversable`.